integreat_cms/release_notes/current/unreleased/1845.yml

```
@@ -0,0 +1,2 @@
+en: Add overall word count of German contents
+de: Füge Gesamtwortzahl der deutschen Inhalte hinzu
```

```suggestion
en: Display of the total number of words of the content in the default language in machine translation management
de: Zeige die Gesamtzahl der Wörter des Inhalts in der Standardsprache in der maschinellen Übersetzungsverwaltung an
```
Not sure, maybe be more specific where the new table is located?

---

sounds good :)